contracts/cw1-subkeys/src/contract.rs

```diff
@@ -1184,7 +1181,7 @@ mod tests {
         let res = execute(deps.as_mut(), mock_env(), info, execute_msg.clone());
         match res.unwrap_err() {
             ContractError::Std(StdError::Underflow { .. }) => {}
-            e => panic!("unexpected error: {}", e),
+            e => panic!("unexpected error: {:?}", e),
```

Comment:
Ah, I remember, the StdError with backtraces cannot be compared with the `assert_eq!` macro. I couldn't figure a way to make this easier when migrating other contracts, but it would be nice to have this shorter as well.

I wonder if you could use [`matches!`](https://doc.rust-lang.org/std/macro.matches.html) to make this easier?

`assert!(matches!(res.unwrap_err(), ContractError::Std(StdError::Underflow { .. }))` or something like that?